src/api/resolvers/models/currencie.ts

```diff
@@ -11,3 +11,6 @@ export interface Currencie {
   changePercent24Hr: string;
   vwap24Hr: string;
 }
+export interface Price {
+  [id: string]: string;
+}
```

Comment:
I think It's easier to change data: any because you don't know what is the real signature